src/pages/member/NewMemberRegister.js

```diff
@@ -26,9 +30,19 @@ const useStyles = makeStyles(theme => ({
   }
 }));
 
-const EventFormContainer = (props) => {
+const NewMemberRegisterFormContainer = (props) => {
   const classes = useStyles();
-  const { event } = props;
+
+  const history = useHistory();
+  const useQuery = () => { return new URLSearchParams(useLocation().search) };
+  const queries = useQuery();
+  
+  // Destructure query url
+  const initialEmail = queries.get('email');
+  let initialName = queries.get('name');
```

Comment:
DOn't think we'll have their name before they sign up, only their email

Reply:
It seems google auth passes a name attribute too (only if available, i think). For facebook auth, will have to check - but even if no name is passed, I made sure there shouldn't be errors popping up.